packages/react-experiments/src/components/FloatingSuggestionsComposite/FloatingSuggestionsList/FloatingSuggestionsList.tsx

```diff
@@ -29,7 +67,39 @@ export const FloatingSuggestionsList = <T extends {}>(props: IFloatingSuggestion
   };
 
   const renderFooter = (): JSX.Element | null => {
-    const { onRenderFooter } = props;
+    const { onRenderFooter, footerItemsProps, selectedFooterIndex, suggestionsFooterContainerAriaLabel } = props;
+
+    if (footerItemsProps) {
+      return (
+        <div
+          className={css('ms-Suggestions-footerContainer', classNames.suggestionsContainer)}
+          id="suggestionFooter-list"
+          role="list"
+          aria-label={suggestionsFooterContainerAriaLabel}
+        >
+          {footerItemsProps.map((footerItemProps: ISuggestionsHeaderFooterProps, index: number) => {
+            const isSelected = selectedFooterIndex !== -1 && selectedFooterIndex === index;
+            return footerItemProps.shouldShow() ? (
+              <div
+                id={'sug-footer' + index}
+                key={'sug-footer' + index}
+                role="listitem"
+                aria-label={footerItemProps.ariaLabel}
+              >
+                <SuggestionsHeaderFooterItem
+                  id={'sug-footer-item' + index}
+                  isSelected={isSelected}
+                  renderItem={footerItemProps.renderItem}
+                  onExecute={footerItemProps.onExecute}
+                  className={footerItemProps.className}
+                />
+              </div>
+            ) : null;
+          })}
+        </div>
+      );
+    }
+
     if (onRenderFooter) {
```

Comment:
same question here for the footer props check. OnRenderFooter will take precedence, if the user decided to provide both footerItemsProps and onRenderFooter